techmatters/flex-plugins
# CHI-1530 tf ssm params refactor

## Description
This sets up ssm params in the new format. The following command has been run for `teguio-staging` and `zimbabwe-staging` for use by the kamailio setup. These commands will need to be run for all other helplines when this is merged:

```bash
make manage-ssm-secrets apply tf_args="-target=module.aws.aws_ssm_parameter.twilio_auth_token -target=module.aws.aws_ssm_parameter.serverless_base_url -target=module.aws.aws_ssm_parameter.s3_docs_bucket_id"
```

This will create the immediately required ssm params in the new format. We will need to get a full apply running without changes for all helplines soon, but this will work as a stop gap and not create more resources that need to be imported/moved when we refactor the terraform layout for twilio-iac

This should be merged before #1145

### Checklist

### Related Issues
#CHI-1632 and #CHI-1530

### Verification steps
Run this from a helpline and ensure the appropriate ssm params are created:

```bash
make manage-ssm-secrets apply tf_args="-target=module.aws.aws_ssm_parameter.twilio_auth_token -target=module.aws.aws_ssm_parameter.serverless_base_url -target=module.aws.aws_ssm_parameter.s3_docs_bucket_id"
```
